Documentation/FrameworkOverview.md

```diff
@@ -8,196 +8,219 @@ learning about new modules and finding more specific documentation.
 For examples and help understanding how to use RAC, see the [README][] or
 the [Design Guidelines][].
 
-## Streams
+## Events
 
-A **stream**, represented by the [RACStream][] abstract class, is any series of
-object values.
+An **event**, represented by the [Event][] class, is the formalized representation
+of the fact that _something has happened_. In Reactive Cocoa, events are the center
+piece of communication. An event might represent the press of a button, a piece 
+of information received from an API, the occurrence of an error or the completion 
+of a long running operation. A source generates events and sends them over a 
+[signal](#signals) to any number of [subscribers](#subscribers).
 
-Values may be available immediately or in the future, but must be retrieved
-sequentially. There is no way to retrieve the second value of a stream without
-evaluating or waiting for the first value.
-
-Streams are [monads][]. Among other things, this allows complex operations to be
-built on a few basic primitives (`-bind:` in particular). [RACStream][] also
-implements the equivalent of the [Monoid][] and [MonadZip][] typeclasses from
-[Haskell][].
-
-[RACStream][] isn't terribly useful on its own. Most streams are treated as
-[signals](#signals) or [sequences](#sequences) instead.
+An event has two types associated with it: the type of the value it holds when 
+everyting goes right, and the type of error it may hold in the case of a failure. 
+Additional to the _normal_ and the _failure_ case, an event can also represent 
+_completion_ and _interruption_.
```

Comment:
Alright. I've cut out the _Terminating events are not included in the streams values, they must be 
handeled specially._ sentences from the bulleted list here though and added it back to the Signals section because I felt that talking about the lifetime of signals, before really introducing them, was too early.